tests/functional/spec/numbers.spec.js

```diff
@@ -13,6 +13,20 @@ describe("Number validation", () => {
     await browser.openQuestionnaire("test_numbers.json");
   });
   describe("Given I am completing the test numbers questionnaire,", () => {
+    it("When a maximum value with decimals is used, Then the correct error message including the minimum value as a float is included", async () => {
```

Comment:
Are these tests not the wrong way around, the minimum is testing maximum and vice versa?.

I think it would help readability if they read something like the following:
`When a maximum value with decimals is used and I enter a value greater than the maximum, Then the error message display the maximum value I can enter including the decimal`

Would need to change for both.

Reply:
They were the wrong way around, thanks and I've updated them 👍